src/spatial/controllers/controllers.py

```
@@ -78,4 +79,26 @@ def fetch_air_quality_without_llm():
+
+@controller_bp.route('/get-data-by-confidence', methods=['GET'])
+def get_data_by_confidence():
+    return PollutantApis.get_data_by_confidence()
+
+@controller_bp.route('/get-all-data', methods=['GET'])
+def get_all_data():
+    return PollutantApis.get_all_data()
+
+
+@controller_bp.route('/get-data-by-confidence', methods=['GET'])
+def get_data_by_confidence():
```

Ruff (0.8.2)

Redefinition of unused `get_data_by_confidence` from line 90

(F811)